src/test/java/org/example/StudRepoTest.java

```diff
@@ -0,0 +1,76 @@
+package org.example;
+import java.util.*;
+import java.time.LocalDate;
+import static org.junit.jupiter.api.Assertions.*;
+import org.junit.jupiter.api.function.Executable;
+class StudRepoTest {
+    @org.junit.jupiter.api.Test
+    void givenValidStudentExpectedListSize() throws ValidationException {
+        StudRepo testRepository = new StudRepo();
+        List<Student> studentList = new ArrayList<>();
+        assertEquals(1, (testRepository.addStudent("And", "He", LocalDate.of(1995, 7, 12), "F", "1920216419912")).size());
+
+    }
+    @org.junit.jupiter.api.Test
+    void givenEmptyFirstNameExpectedException() throws ValidationException {
+        Exception exception = assertThrows(ValidationException.class, new Executable() {
```

Comment:
you can just use the lambda expresion we showed in class () -> testRepository.addStudent()